### PR TITLE
wats ur fav type of icecream

### PR DESCRIPTION
**Pull request by demipixel**
Post in the comments!

### CODE REVIEW

**Comment by rom1504** [comment]
Circle ci points out a few errors, look at it ;)

**Comment by 109C** [comment]
Chocolate and vanilla.

File: doc/CONTRIBUTE.md [hunk 1]
```diff
@@ -5,93 +5,67 @@
...
+  player.on(',,,', ...);
+  // serv.on('...', .– don't even think about it
 }
+
 ```
 
 ## Logs and event
 
 In order to keep logging independent from the rest of the server and to let people react in other ways than logging,
-server and player events should be emitting and the logging should only take place in response to these events
```
Comment (rom1504) [inline doc/CONTRIBUTE.md]: that's still true

**Comment by rom1504** [comment]
when you do `/help command`, the description of the command is displayed 2 times, that's one time too much

**Comment by demipixel** [comment]
will not commit unless u post ur fav ice cream

**Comment by rom1504** [comment]
secondly, why did you remove the usage from the main `/help` ? vanilla has it. (and I think it's useful)

hmm I guess my favorite ice cream might be vanilla, or mango :p

File: doc/CONTRIBUTE.md (hunk 1 above)
Comment (demipixel) [inline doc/CONTRIBUTE.md]: serv.log() isn't an event. That's why I added

    logging uses methods and event from `log.js` These include...

Comment (rom1504) [inline doc/CONTRIBUTE.md]: yes, but what I meant is player and server events are emitted, and then listened to in log.js which call serv.log

**Pull request merged**